pom.xml

```diff
@@ -90,9 +90,13 @@
     <logback.version>1.2.3</logback.version>
     <maven.project.version>2.2.1</maven.project.version>
     <mockito.version>4.11.0</mockito.version>
-    <sonar.version>9.9.1.69595</sonar.version>
+    <!--
+      sonar-plugin-api should be compatible with SQ peach. You can see which version SQ is using
+      at SonarSource/sonarqube/gradle.properties
+    -->
+    <sonar.version>10.6.0.92116</sonar.version>
+    <sonar.api.version>10.7.0.2191</sonar.api.version>
     <sonar.orchestrator.version>5.0.0.2065</sonar.orchestrator.version>
```

Comment:
Update was needed because in the previous version, `DefaultInputFile#md5hash()` was not implemented